setup.py

```diff
@@ -14,10 +25,10 @@
    author='Yu-Xiang Wang',
    author_email='yuxiangw@cs.ucsb.edu',
    url='https://github.com/yuxiangw/autodp',
-   download_url = 'https://github.com/yuxiangw/autodp/archive/v0.1.1.tar.gz',   
+   download_url = 'https://github.com/yuxiangw/autodp/archive/v0.1.1.tar.gz',
```

Comment:
This has changed to v0.2. I will see if I can directly edit  setup.py in your branch.